app/components/collectionCardCount.tsx

```diff
@@ -1,8 +1,14 @@
-import React from 'react';
+import React, { useState, useEffect } from "react";
 import './../../styles/collection.css';
 
 export default function CollectionCardCount(props: {count: string}) {
+    const [count, setCount] = useState("");
```

Comment:
This could probably be a number

Reply:
I remember why i made it a string now. That way on initial load it shows nothing instead of a number. Otherwise, it will show a 0 on the screen, basically baiting the user into thinking it was 0, instead of it just not being loaded yet.